protocol/clproto_cpp/src/clproto.cpp

```diff
@@ -773,6 +773,50 @@ static bool decode_parameter(const std::string& msg, Parameter<T>& obj) {
   }
 }
 
+/* ----------------------
+ *        INT
```

Comment:
maybe give this and the one below two more spaces on the left side 😁 
```suggestion
 *          INT
```